ui/packages/catalog-ui-search/src/main/webapp/component/query-src/query-src.less

```diff
@@ -1,16 +1,29 @@
 @{customElementNamespace}query-src {
```

Comment:
If we update the handlebars template for this component to be:
```
{{!--
/**
 * Copyright (c) Codice Foundation
 *
 * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
 * version 3 of the License, or any later version.
 *
 * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
 * <http://www.gnu.org/licenses/lgpl.html>.
 *
 **/
 --}}
<div class="choice is-all is-available">
    <span class="choice-text">All Sources</span>
</div>
{{#each this}}
    <div class="choice is-specific {{#if available}}is-available{{/if}}" data-value="{{id}}">
        <span class="choice-text">
            {{#if available}}
                <span class="fa fa-exclamation-triangle">

                </span>
            {{/if}}
            {{#if local}}
                <span class="fa source-icon fa-home">
                </span>
            {{else}}
                <span class="fa source-icon fa-cloud">
                </span>
            {{/if}}
            {{id}}
        </span>
    </div>
{{/each}}
```

Then we could use this to accomplish what we want in the css:
```
@{customElementNamespace}query-src {
  display: block;

  .choice {
    padding: 0px @minimumSpacing;
    height: @minimumButtonSize;
    line-height: @minimumButtonSize;
    cursor: pointer;
    position: relative;
  }

  .choice-text {
    display: inline-block;
    overflow: hidden;
    white-space: nowrap;
    text-overflow: ellipsis;
    position: relative;
    max-width: ~'calc(100% - @{minimumButtonSize})';
  }

  .choice-selected {
    line-height: @minimumButtonSize;
    display: none;
  }

  .choice.is-selected::after {
    display: inline-block;
    content: "\f00c";
    font-family: FontAwesome;
    font-style: normal;
    position: absolute;
    top: 50%;
    transform: translateY(-50%);
    right: 0px;
    width: @minimumButtonSize;
    text-align: center;
  }

  .choice.is-selected .choice-text{
    .is-bold();
  }

  .choice:not(.is-available) {
    opacity: @minimumOpacity;
  }

}
```

Basically, move the icons into the text span and then we only need to make room for the possible checkmark on the end.